website/docs/d/bucket.html.markdown

```diff
@@ -0,0 +1,39 @@
+---
+layout: "runscope"
+page_title: "Runscope: runscope_bucket"
+sidebar_current: "docs-runscope-datasource-bucket"
+description: |-
+  Get information about a single runscope bucket.
+---
+
+# runscope\_bucket
+
+Use this data source to get information about a specific [bucket](https://www.runscope.com/docs/api/buckets)
+that you can with other runscope resources.
+
```

Comment:
type missing word `use`

 "that you can with other" assume it should be "that you can use with other"